docker/release/cudaq.ext.Dockerfile

```diff
@@ -6,7 +6,7 @@
 # the terms of the Apache License 2.0 which accompanies this distribution.     #
 # ============================================================================ #
 
-ARG base_image=ghcr.io/nvidia/cuda-quantum:latest
+ARG base_image=ghcr.io/nvidia/cuda-quantum:latest-base
```

Comment:
Just for my own benefit, what's the difference between the `latest` and `latest-base` tags? Does `latest-base` include all the cuda-quantum stuff in it, or is it just the base `cuda` image?

Reply:
latest-base contains all components in this repo, but not the closed source GPU-based simulator backends.